extensions/wikia/ExactTargetUpdates/client/adapters/UserEditsAdapter.class.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Wikia\ExactTarget;
+
+class UserEditsAdapter {
+
+	var $edits = [ ];
+
+	public function __construct( $result ) {
+		if ( $result->Properties instanceof \stdClass ) {
+			$this->extractUserWikiEdits( $result->Properties->Property );
```

Comment:
At this place we check is Properties has something, and after that we can run `->Property` on null.
Is instance of is a good enough check for that?